index.html

```diff
@@ -14,252 +14,400 @@
+              <li>
+                <a href=""
+                  ><span class="iconify" data-icon="iconoir:github"></span>
+                </a>
+              </li>
+              <li>
+                <a href=""
+                  ><span class="iconify" data-icon="eva:linkedin-fill"></span>
+                </a>
+              </li>
+              <li>
+                <a href=""
+                  ><span class="iconify" data-icon="fontisto:angelist"></span>
+                </a>
+              </li>
+              <li>
+                <a href=""
+                  ><span class="iconify" data-icon="ei:sc-twitter"></span>
+                </a>
+              </li>
+              <li>
+                <a href=""
+                  ><span class="iconify" data-icon="bx:bxl-medium"></span>
+                </a>
+              </li>
+            </ul>
+          </div>
         </div>
-        <div class="let-section">
-          <h4 class="connect">LET’S CONNECT</h4>
-          <img src="img/Ellipse 12.png" alt="Eclipse-image" />
-        </div>
-
-        <div class="eclipse-13">
-          <img src="img/Ellipse 13.png" alt="Eclipse-image" />
-        </div>
+      </section>
+    </header>
 
-        <ul class="social-icons">
-          <li><span class="iconify" data-icon="iconoir:github"></span></li>
-          <li><span class="iconify" data-icon="eva:linkedin-fill"></span></li>
-          <li><span class="iconify" data-icon="fontisto:angelist"></span></li>
-          <li><span class="iconify" data-icon="ei:sc-twitter"></span></li>
-          <li><span class="iconify" data-icon="bx:bxl-medium"></span></li>
+    <main class="main-section">
+      <section class="tonic-section">
+        <ul class="images">
+          <li>
+            <img
+              src="img/card Portfolio.png"
+              alt="card"
+              class="desktop-images"
+            />
+          </li>
+          <li>
+            <img
+              src="img/Snapshoot Portfolio.png"
+              alt="card"
+              class="mobile-images"
+            />
+          </li>
         </ul>
 
-        <div class="eclipse-4">
-          <img src="img/Ellipse 4.png" alt="Eclipse-image" />
-          <img src="img/Rectangle 5.png" alt="Eclipse-image" />
+        <div class="tonic">
+          <h3>Tonic</h3>
+          <div class="date">
+            <h5>CANOPY</h5>
+            <img src="img/Counter.png" alt="counter" class="counter" />
+            <p>Back End Dev</p>
+            <img src="img/Counter.png" alt="counter" class="counter" />
+            <p>2015</p>
+          </div>
+          <p class="daily">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <div>
+            <button>html</button>
+            <button>css</button>
+            <button class="javascript">javaScript</button>
+          </div>
+          <button class="see-project">See Project</button>
```

Comment:
### OPTIONAL

- [ ] `See Project `button should take you to the Live Demo of the project or to the GitHub repository. You can fix it.